feat: Adding screenshot (#30597)
github/docs: content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
{% data reusables.repositories.allow-maintainers-user-forks %}

> ![Screenshot of a Pull request. On the right bottom side, a blue, clicked checkbox, labelled "Allow edits and access to secrets by maintainers".](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)

Reviewer: The ! sign must be aligned with the first word in the bullet point (which is "In...the list", so that the ALT text and the screenshot are at the same level as the step or bullet point they relate to.
Also, removing the capitalization of "Pull request" and using US spelling.
Finally, added `is highlighted with an orange outline.` to be consistent with the content model we use.

```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, clicked checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

> ![Screenshot of a Pull request. On the right bottom side, a blue, clicked checkbox, labelled "Allow edits and access to secrets by maintainers".](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)

Reviewer: Our images or screenshots must be taken with the GitHub default light theme on (for more details, see https://github.com/settings/appearance. You'll need to select "Light default" before taking the screenshot), and also the orange outline has a specific line thickness and a specific round corner measurement.
We may need to provide you with the screenshot ourselves. Let me investigate.

>    ![Screenshot of a pull request. On the right bottom side, a blue, clicked checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)

Reviewer: ```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

>    ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)

Reviewer: Changing the name of the screenshot as per our content model.
```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)
```

>    ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)

Reviewer: Adding missing comma
```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers", is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)
```
